clouddriver-saga/src/main/kotlin/com/netflix/spinnaker/clouddriver/saga/models/Saga.kt

```diff
@@ -68,6 +78,20 @@ class Saga(
   var compensating: Boolean = false
```

Comment:
Wondering if we should add a default compensate in SagaEventHandler and have that one just turn the flag to true. Then have impl have to call `super.compensate`. Has to be a better way

Reply:
What do you mean by "default compensate"? How could there be a generic default?